<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewProjectDialog</class>
 <widget class="QDialog" name="NewProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Project</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Project Settings</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelName">
        <property name="text">
         <string>Project Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="editProjectName"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelLocation">
        <property name="text">
         <string>Location:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="locationLayout">
        <item>
         <widget class="QLineEdit" name="editLocation"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnBrowse">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelLanguage">
        <property name="text">
         <string>Language:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboLanguage">
        <item>
         <property name="text">
          <string>C++</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>C</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelStandard">
        <property name="text">
         <string>Standard:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboStandard">
        <item>
         <property name="text">
          <string>C++20</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>C++17</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>C++14</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>C11</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>C14</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>C17</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>C20</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupProjectType">
     <property name="title">
      <string>Project Type</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioConsole">
        <property name="text">
         <string>Console Application</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkGraphics">
        <property name="text">
         <string>Enable graphics.h (32-bit, legacy)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkOpenMP">
        <property name="text">
         <string>Enable OpenMP (multi-threading)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioGraphics">
        <property name="text">
         <string>Graphics Application (graphics.h - 32-bit only)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewProjectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewProjectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
